### PR TITLE
TM-679: Some links in mail can not open in other tabs

### PR DESCRIPTION
**Pull request by ManhNTX**
https://user-images.githubusercontent.com/107173849/177191314-89b9b3aa-b52b-4f3f-9f22-fd535b03f6da.mov

### CODE REVIEW

File: core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart [hunk 1]
```diff
@@ -104,8 +104,6 @@ class _HtmlContentViewerOnWebState extends State<HtmlContentViewerOnWeb> {
            } else if (textContent && isValidUrl(textContent)) {
               window.parent.postMessage(JSON.stringify({"view": "$createdViewId", "type": "toDart: OpenLink", "url": "" + textContent}), "*");
               e.preventDefault();
-           } else {
-              e.preventDefault();
```
Comment (dab246) [inline core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart:108]: Could you please explain more about this change?

Comment (ManhNTX) [reply core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart:108]: When you click tag <a/> of HTML, function onClickListener cant handle tag <a/>, this function will handle child of tag <a/>. So code will run to line 107, but it this line you set `e.preventDefault()`. It reason no action will cotinue when click.

Comment (dab246) [reply core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart:108]: @ManhNTX

I've fetched your code up and running and some emails can't open in a new tab. Please try to double check with emails with website content more closely. See video below for details.

https://user-images.githubusercontent.com/80730648/177456238-daadeefb-85f8-4e6d-82fb-be1c7c0b7181.mov

Comment (dab246) [reply core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart:108]: This error appears on `https://jmap.linagora.com` but on `https://gateway.upn.integration-open-paas.org` not.

**Review by dab246 — APPROVED** [review]

**Pull request merged**